haas/cli.py

```diff
@@ -695,7 +695,11 @@ def show_network(network):
 @cmd
 def show_node(node):
     """Display information about a <node>
+
     FIXME: Recursion should be implemented to the output.
+    The output of show_node is a dictionary that can be list of list, having
+    multiple nics and networks. More meta data about node could be shown
+    via this call. So this is a note for developers that will work on CLI.
```

Comment:
Could you move this to a comment just below? the docstring gets displayed to the user of cli tool, which isn't really appropriate for this information.